supervisor/dbus/network/__init__.py

```diff
@@ -23,6 +30,8 @@
 
 _LOGGER: logging.Logger = logging.getLogger(__name__)
 
+MINIMAL_VERSION = "1.14.0"
```

Comment:
```suggestion
MINIMAL_VERSION = "1.14.6"
```
<https://github.com/home-assistant/architecture/blob/master/adr/0014-home-assistant-supervised.md#supported-operating-system-system-dependencies-and-versions>